### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by v-fearam**
1- OMS agent
 Updates following the referenced  file 
2- Preview feature dependencies

### CODE REVIEW

File: 01-prerequisites.md [hunk 1]
```diff
@@ -28,14 +28,11 @@ This is the starting point for the instructions on deploying the [AKS Baseline r
...
    1. [Register the Azure Event Grid preview feature - `EventgridPreview`](https://docs.microsoft.com/azure/aks/quickstart-event-grid#register-the-eventgridpreview-preview-feature)
 
-   1. [Register the Kubernetes Configuration preview feature - `fluxConfigurations`](https://docs.microsoft.com/azure/azure-arc/kubernetes/tutorial-use-gitops-flux2#for-azure-kubernetes-service-clusters)
-
    1. [Register the OIDC Issuer preview feature = `EnableOIDCIssuerPreview`](https://docs.microsoft.com/azure/aks/cluster-configuration#oidc-issuer-preview)
 
    ```bash
    az feature register --namespace "Microsoft.ContainerService" -n "EventgridPreview"
    az feature register --namespace "Microsoft.ContainerService" -n "AKS-ExtensionManager"
-   az feature register --namespace "Microsoft.KubernetesConfiguration" -n "fluxConfigurations"
```
Comment (ckittel) [inline 01-prerequisites.md:38]: and on line 40.

File: cluster-manifests/kube-system/container-azm-ms-agentconfig.yaml [hunk 2]
```diff
@@ -1,45 +1,164 @@
-apiVersion: v1
 kind: ConfigMap
-metadata:
-  name: container-azm-ms-agentconfig
-  namespace: kube-system
```
Comment (ckittel) [inline cluster-manifests/kube-system/container-azm-ms-agentconfig.yaml:5]: Please keep the metadata at the top.  I know it's at the bottom in the other, but that's unstandard for a yaml manifest.

**Review by ckittel — CHANGES_REQUESTED** [review]

File: 01-prerequisites.md (hunk 1 above)
Comment (v-fearam) [reply 01-prerequisites.md:38]: done

File: cluster-manifests/kube-system/container-azm-ms-agentconfig.yaml (hunk 2 above)
Comment (v-fearam) [reply cluster-manifests/kube-system/container-azm-ms-agentconfig.yaml]: done

File: 01-prerequisites.md [hunk 3]
```diff
@@ -34,7 +34,6 @@ This is the starting point for the instructions on deploying the [AKS Baseline r
    az feature register --namespace "Microsoft.ContainerService" -n "EventgridPreview"
    az feature register --namespace "Microsoft.ContainerService" -n "AKS-ExtensionManager"
    az feature register --namespace "Microsoft.ContainerService" -n "EnableOIDCIssuerPreview"
-   az feature register --namespace "Microsoft.ContainerService" -n "AKS-AzureDefender"
```
Comment (ckittel) [inline 01-prerequisites.md:40]: Why did you do this?

**Review by ckittel — CHANGES_REQUESTED** [review]

File: cluster-manifests/kube-system/container-azm-ms-agentconfig.yaml (hunk 2 above)
Comment (ckittel) [reply cluster-manifests/kube-system/container-azm-ms-agentconfig.yaml]: Thanks.

File: 01-prerequisites.md (hunk 1 above)
Comment (ckittel) [reply 01-prerequisites.md:38]: you deleted something, but wasn't the item requested that we talked about in our call or here in the thread.  Why did you get rid of "Microsoft.ContainerService" -n "AKS-AzureDefender"?

File: 01-prerequisites.md (hunk 3 above)
Comment (v-fearam) [reply 01-prerequisites.md:40]: Sorry, I misunderstood

**Review by ckittel — APPROVED** [review]
LGTM!  :shipit:

**Pull request merged**